### PR TITLE
run registry/image tests on appropriate changes

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@minimar @kargakis ptal

File: cluster/ci/config/prow/config.yaml [hunk 1]
```diff
@@ -165,6 +165,7 @@ presubmits:
   - name: test_pull_request_origin_extended_image_registry
     agent: jenkins
     always_run: false
+    run_if_changed: '^(pkg/image)|^(test/extended/registry)|^(test/extended/images)|^(test/extended/imageapis)|^(vendor/github.com/docker/distribution)'
```
Comment (kargakis) [inline cluster/ci/config/prow/config.yaml]: Do you also want `/test/extended/image_ecosystem`?

**Comment by kargakis** [comment]
oh, that's the change you wanted in. sorry. 

I had one question

File: cluster/ci/config/prow/config.yaml (hunk 1 above)
Comment (bparees) [reply cluster/ci/config/prow/config.yaml]: no, that's a different test job.

**Pull request merged**